web-api/migration-cron-terraform/bin/destroy-app.sh

```diff
@@ -41,6 +41,9 @@ export TF_VAR_migrate_flag=$MIGRATE_FLAG
 export TF_VAR_source_table=$SOURCE_TABLE
 export TF_VAR_stream_arn=$STREAM_ARN
 
+mkdir -p ./lambdas/dist
+touch ./lambdas/dist/reindex-status.js
```

Comment:
Why do we want to make that empty file?

Reply:
for some reason, terraform expects those files to exist locally for it to successfully run the terraform destroy.    More specifically this code fails saying "file does not exist"

```
// web-api/migration-cron-terraform/main/reindex-status.tf
data "archive_file" "reindex_status_zip" {
  type        = "zip"
  output_path = "${path.module}/lambdas/reindex-status.js.zip"
  source_dir  = "${path.module}/lambdas/dist/"
  excludes = ["migration.js", "migration-segments.js"]
}
```

maybe there is a way to not require the file to exist when doing a tear down?  since it's a `data` resource I guess it always needs to exist.